pkg/api/v1/client/client.go

```diff
@@ -83,3 +83,10 @@ func (c *Client) ServerConditionCreate(ctx context.Context, serverID uuid.UUID,
 
 	return c.post(ctx, path, conditionCreate)
 }
+
+func (c *Client) ServerEnroll(ctx context.Context, serverID string, conditionCreate v1types.ConditionCreate) (*v1types.ServerResponse, error) {
```

Comment:
Question for thought: would it be easier/better to break this out into a method with more "primitive" arguments (i.e. strings or golang std library types)?